### PR TITLE
vnstat: Update to 2.1

### PR DESCRIPTION
**Pull request by neheb**
Reorganized Makefile for consistency between packages.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @jow- 
Compile tested: ramips

### CODE REVIEW

**Comment by thess** [comment]
CCI fixed - please rebase to pick up script update.

File: net/vnstat/Makefile [hunk 1]
```diff
@@ -26,12 +28,13 @@ include $(INCLUDE_DIR)/package.mk
...
   SECTION:=net
   CATEGORY:=Network
-  URL:=http://humdi.net/vnstat/
+  URL:=https://humdi.net/vnstat/
 endef
 
 define Package/vnstat
 $(call Package/vnstat/Default)
   TITLE:=Console-based network traffic monitor
+  DEPENDS:=+libsqlite3
```
Comment (cshoredaniel) [inline net/vnstat/Makefile]: Please don't add major new dependencies and use a **misleading** commit message like "Reorganize Makefile..." that makes it look like just housekeeping.

**Comment by jow-** [comment]
NACK on this change. I'm fine with adding vnstat 2 (with another maintainer) but for 1.x I'd like to avoid the sqlite3 dependency.

**Comment by neheb** [comment]
Ok

**Pull request closed without merging**